desktop/models/sale.coffee

```diff
@@ -39,6 +39,8 @@ module.exports = class Sale extends Backbone.Model
 
   bidUrl: (artwork) ->
     "#{@href()}/bid/#{artwork.id}"
+  
+  predictionUrl: -> "#{PREDICTION_URL}/#{@get('slug')}"
```

Comment:
Can you just use `{@id}` here?

Also #minor but I'd call this `liveAuctionUrl` as we do elsewhere.